lighthouse-core/lib/network-recorder.js

```diff
@@ -34,6 +38,10 @@ class NetworkRecorder extends EventEmitter {
     );
   }
 
+  getRecords() {
+    return this._records.slice();
```

Comment:
Array.from() ?

Reply:
haha, man between you and dgozman I'm gonna lose myself! 😆 

done